Defs/Ammo/Advanced/80x256mmFuelCell.xml

```diff
@@ -49,9 +49,10 @@
 	<detonateProjectile>Bullet_80x256mmFuel_Thermobaric</detonateProjectile>
 	<comps>
     <li Class="CompProperties_Explosive">
-        <explosiveRadius>2</explosiveRadius>
+        <explosiveRadius>1</explosiveRadius>
+		<damageAmountBase>20</damageAmountBase>
         <explosiveDamageType>Thermobaric</explosiveDamageType>
-		<explosiveExpandPerStackcount>0.15</explosiveExpandPerStackcount>
+		<explosiveExpandPerStackcount>0.10</explosiveExpandPerStackcount>
```

Comment:
Just wondering if this is the line preventing chain reactions?

Reply:
This or adding the base damage amount, pretty sure.